frontend/packages/dev-console/src/components/monitoring/dashboard/monitoringDashboardQueries.ts

```diff
@@ -0,0 +1,47 @@
+import { template } from 'lodash';
+import {
+  humanizeBinaryBytes,
+  humanizeCpuCores,
+  humanizeDecimalBytesPerSec,
+} from '@console/internal/components/utils';
+import { ByteDataTypes } from '@console/shared/src/graph-helper/data-utils';
+import { GraphTypes } from './MonitoringDashboardGraph';
+
+export const queries = [
```

Comment:
Perhaps give this a type so if any other items are added to the array they conform to the same shape.